packages/server/src/healthcheck.ts

```diff
@@ -1,16 +1,30 @@
 import { MEDPLUM_VERSION } from '@medplum/core';
 import { Request, Response } from 'express';
 import { getDatabasePool } from './database';
+import { setGauge } from './otel/otel';
 import { getRedis } from './redis';
 
 export async function healthcheckHandler(_req: Request, res: Response): Promise<void> {
+  setGauge('medplum.db.idleConnections', getDatabasePool().idleCount);
+  setGauge('medplum.db.queriesAwaitingClient', getDatabasePool().waitingCount);
+
+  let startTime = Date.now();
+  const postgresOk = await testPostgres();
+  const dbRoundtripMs = Date.now() - startTime;
+  setGauge('medplum.db.roundtripMs', dbRoundtripMs);
+
+  startTime = Date.now();
+  const redisOk = await testRedis();
+  const redisRoundtripMs = Date.now() - startTime;
+  setGauge('medplum.redis.roundtripMs', redisRoundtripMs);
```

Comment:
For consistency, we might want to report all time metrics like this in whole seconds

Reply:
Since this metric is almost always going to be sub ~100ms, isn't it easier to read in milliseconds? or is there some kind of convention on reporting time in OTel?

Reply:
It's a convention for metrics reporting, since the observability frontend is usually responsible for displaying the data points with whatever sub-unit (e.g. `s`, `ms` or `µs`) is appropriate